mlir/tools/mlir-miopen-lib/mlir-miopen-lib.cpp

```diff
@@ -55,6 +55,28 @@ static void strToTokens(const std::string &arguments,
     }
   }
 }
+
+// In multi-threaded context, static intialization is guaranteed to
+// be thread safe and atomic. With this guarantee, we are protected
+// from the possible race condition of one thread doing intialization
+// and another doing lowering.
+bool lazy_init() {
```

Comment:
:trollface: lazy_init is used by lazy programmers who are too lazy to rename functions according to their own coding style.  ; )

Reply:
:sweat_smile: :sweat_smile:  :sweat_smile: 

Will rename it to `miirLazyInit()`